src/pytest_inmanta_lsm/plugin.py

```diff
@@ -234,37 +232,50 @@ def remote_orchestrator_shared(
                     "The module being tested is a v1 module but it is also installed as a v2 module. Local compiles will use"
                     "the v2, but only the v1 will by synced to the server."
                 )
+
+        elif installed is None:
+            # Make sure that the module v2 source can be found, it should always be the case, unless
+            # this fixture is used outside of the context of a module test suite.
+            raise RuntimeError(f"Module {mod.name}'s source could not be found")
```

Comment:
This scenario is a nice catch, looks like an oversight in the initial implementation. But I think a more appropriate error message would be that the module is not installed, this informs the user how to resolve it. Honestly I'd just pack this condition together with the next one and raise the same error, it's accurate enough: not installed => not installed in editable mode.

Reply:
I refactored a bit the install check for modules v2, I am raising a common exception, and still keeping separate branches with different error logs as it has already helped me debugging what was going on in the test cases of this project

Reply:
Ok, in that case I do think we should make the error message more useful to the end user though (you may have already done so, haven't checked your latest changes yet).